raiden/tests/integration/contracts/test_payment_channel.py

```diff
@@ -97,7 +97,7 @@ def test_payment_channel_proxy_basics(
     # balance proof by c2
     transferred_amount = 3
     balance_proof = BalanceProof(
-        channel_identifier=encode_hex(channel_identifier),
```

Comment:
Don't touch the channel identifier at all please. This is coming in my PR which turns it from bytes32 to uint256 and if you make changes on same code I am touching we will end up having conflicts.

Reply:
Same.